lib/cr-base.js

```diff
@@ -6,18 +6,18 @@ var Debugger = function(prefixStr) {
   this.debug = {};
   if (atom.devMode || force_atom_devMode) {
     for (var m in console)
-      if (typeof console[m] == 'function')
-        this.debug[m] = console[m].bind(window.console, prefixStr);
+      if (typeof console[m] == 'function') // eslint-disable-line no-console
+        this.debug[m] = console[m].bind(window.console, prefixStr); // eslint-disable-line no-console
   }
   else {
-    for (var m in console)
-      if (typeof console[m] == 'function')
+    for (var m in console) // eslint-disable-line no-redeclare
+      if (typeof console[m] == 'function') // eslint-disable-line no-console
```

Comment:
well this commenting is annoying for eslint

Reply:
Nah, I'm actually thinking about putting ESlint as a travis build to enforce this kind of thing since it essentially forces developers to explicitly recognize that what they're doing isn't technically correct or best practice.

Reply:
So enforce ESlint in a build for CI in this project? I do not have experience with CI but would be cool to try for me

Reply:
Unless you are referrencing to do it yourself as another project haha, not sure

Reply:
Check out [this](https://medium.freecodecamp.org/how-to-stop-errors-before-they-ever-hit-your-codebase-with-travis-ci-and-eslint-7a5a6b1fcd4a), want me to look into it?